cmake/std/atdm/ATDMDevEnvSettings.cmake

```diff
@@ -313,6 +311,21 @@ IF (ATDM_ENABLE_SPARC_SETTINGS)
   ATDM_SET_CACHE(Anasazi_ENABLE_RBGen ON CACHE BOOL)
 ENDIF()
 
+# Disable/hide deprecated code
+ATDM_SET_CACHE(KOKKOS_ENABLE_DEPRECATED_CODE OFF CACHE BOOL)
```

Comment:
If you made this kokkos flag lower case then this wouldn't this be backwards compatible with the empire scripts?

Reply:
@rppawlo, yes, but then it would conflict with the current SPARC Trilinos configuration scripts :-(  The SPARC Trilinos configuration script scripts used `Kokkos_ENABLE_Deprecated_Code` many months ago and then 3 weeks ago, they switched to `KOKKOS_ENABLE_DEPRECATED_CODE`.  The problem is that Evan copied out the `Kokkos_ENABLE_Deprecated_Code` version before SPARC updated :-(

I created:

* https://cee-gitlab.sandia.gov/EM-Plasma/BuildScripts/merge_requests/262

to match them up.

The other option is to first create a PR that removes `Kokkos_ENABLE_Deprecated_Code` altogether, go back to the old implementation of `EM-Plasma/BuildScripts/build_trilinos.py`, and then merge this PR.  That would need to happen for consecutive days for it to occur smoothly.

I wish the EMPIRE Trilinos sync process allowed for multiple non-backward compatible branches to be merged at the same time in @jmgate's pipeline sync process.  I proposed a process that would allow that to happen.